### DISCUSSION TITLE
Authentication for docker deployments

### DISCUSSION THREAD

**Discussion by hkarakose** [microsoft/presidio#610] [Requested features]
Is there a way to enable authentication for docker based deployments?

**Comment by navalev** [comment]
Hi @hkarakose. We don't support this on presidio level, the platform should provide the authetication process (k8s, App Service on Azure etc.). Is there any specific deployment platform you are targeting?

**Reply by hkarakose** [reply]
I will use it in AWS

**Reply by tomarharsh** [reply]
Looking for something similar. @hkarakose Were you able to add some kind of authentication for your Docker?
